package.json

```diff
@@ -33,7 +33,8 @@
   "dependencies": {
     "flux": "^3.1.2",
     "loglevel": "^1.4.1",
-    "promised-neovim-client": "^2.0.2"
+    "promised-neovim-client": "^2.0.2",
+    "react": "^15.5.4"
```

Comment:
React is not necessary. flux sets react as a peer dependency but we don't use it. So we should ignore the UNMET DEPENDENCY error.